metrics_utility/management/commands/build_report.py

```diff
@@ -61,43 +79,37 @@ def init_logging(self):
         handler.setLevel(logging.DEBUG)
         handler.setFormatter(logging.Formatter('%(message)s'))
         self.logger.addHandler(handler)
-        self.logger.propagate = False
+        self.logger.propagate = True
+
+    def _parse_param(
+        self,
+        param,
+        options,
+    ):
+        if options.get('month') is not None:
+            return None
+
+        if options.get(param):
+            return parse_date_param(options.get(param))
+        else:
+            return None
 
-    def handle(self, *args, **options):
+    def _handle(self, *args, **options):
         self.init_logging()
+        og_month, month, next_month = handle_month(options.get('month') or None)
 
-        # parse params
-        opt_month = options.get('month') or None
-        opt_month, month, next_month = self._handle_month(opt_month)
+        validate_date_extra_params(self, options)
+        opt_month = og_month if options.get('month') else None
 
-        # Since and ephemenral params are specific to subset of reports
-        opt_since = None
-        opt_ephemeral = None
-
-        opt_since = options.get('since') or None
-        opt_since = parse_date_param(opt_since)
-
-        opt_ephemeral = options.get('ephemeral') or None
-
-        opt_until = None  # Initialize opt_until to None
-
-        if opt_since is not None:
-            # If --since was provided, then process --until
-            user_provided_until = options.get('until')
```

Comment:
previously, `opt_until` would not be set (or parsed) unless `opt_since` is as well

but now we're setting both when the param is present .. which changes `_since_until` in ReportCCSPv2 to prefer `until` over `month` .. and `validate_date_extra_params` only warns about that case, but doesn't error out.

Reply:
I'll make it error out.